index.ts

```diff
@@ -99,6 +99,11 @@ export function get_entry(hash: string): string {
   return handleSerialization(jsonEncodedParams, (e: u32): u32 => env.hc_get_entry(e));
 }
 
+export function call(zome_name: string, cap_name: string, fn_name: string, fn_args: string): string {
+  let jsonEncodedParams: string = `{"zome_name":"`+zome_name+`", "cap_name":"`+cap_name+`", "fn_name":"`+fn_name+`", "fn_args":"`+fn_args+`"}`;
```

Comment:
We could use stringify here instead but I actually think I like not depending on that for the core API functions

Reply:
yeah, me too. on that note, we are very happy to have this PR in the works, which will really clarify and simplify the structs for the API serializing, what it expects, and what it gives
https://github.com/holochain/holochain-rust/pull/479/files